pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerTest.java

```diff
@@ -181,9 +181,15 @@ public ControllerRequestClient getControllerRequestClient() {
     return _controllerRequestClient;
   }
 
+  public void startDefaultTestZk() {
+    if (_zookeeperInstance == null) {
+      _zookeeperInstance = ZkStarter.startLocalZkServer(ZkStarter.DEFAULT_ZK_TEST_PORT);
+    }
+  }
+
   public void startZk() {
     if (_zookeeperInstance == null) {
-      _zookeeperInstance = ZkStarter.startLocalZkServer();
+      _zookeeperInstance = ZkStarter.startLocalZkServer(NetUtils.findOpenPort(20000 + RandomUtils.nextInt(10000)));
```

Comment:
any specific reason to find a port between 20000 and 30000?

Reply:
no, just random picked.